### PR TITLE
Removed synthetic accessors

### PR DESCRIPTION
**Pull request by PaulWoitaschek**
Helps people not needing to struggle with multidex.

In AS, there is an inspection under Java/J2ME issues called `Synthetic Accessor call`.

### CODE REVIEW

File: conductor/src/main/java/com/bluelinelabs/conductor/changehandler/SharedElementTransitionChangeHandler.java [hunk 1]
```diff
@@ -155,7 +155,9 @@ public boolean onPreDraw() {
         to.getViewTreeObserver().addOnPreDrawListener(onPreDrawListener);
     }
 
-    private void waitOnChildTransitionNames(@NonNull final View to, @NonNull List<View> foundViews, @NonNull final OnPreDrawListener parentPreDrawListener, @NonNull final OnTransitionPreparedListener onTransitionPreparedListener) {
+    void waitOnChildTransitionNames(
```
Comment (EricKuck) [inline conductor/src/main/java/com/bluelinelabs/conductor/changehandler/SharedElementTransitionChangeHandler.java]: Can you fix the funky linebreak on this?

File: conductor/src/main/java/com/bluelinelabs/conductor/changehandler/AnimatorChangeHandler.java [hunk 2]
```diff
@@ -157,7 +157,9 @@ private void complete(@NonNull ControllerChangeCompletedListener changeListener,
         onAnimationReadyOrAbortedListener = null;
     }
 
-    private void performAnimation(@NonNull final ViewGroup container, @Nullable final View from, @Nullable final View to, final boolean isPush, final boolean toAddedToContainer, @NonNull final ControllerChangeCompletedListener changeListener) {
+    void performAnimation(
```
Comment (EricKuck) [inline conductor/src/main/java/com/bluelinelabs/conductor/changehandler/AnimatorChangeHandler.java]: Can you fix the funky linebreak on this?

**Comment by EricKuck** [comment]
This is great! I made 2 very small requests, but I'm looking forward to merging it. Thanks!

**Comment by PaulWoitaschek** [comment]
Like this?

**Comment by EricKuck** [comment]
Thanks @PaulWoitaschek! This is something I kept meaning to do and never got around to.

**Comment by PaulWoitaschek** [comment]
You're welcome! When building libraries it's really nice to just leave the inspection on so you don't introduce new unwanted methods.

**Pull request merged**